src/testSetup.ts

```diff
@@ -961,6 +961,7 @@ export class MockTestOrgData {
   public isDevHub?: boolean;
   public isScratchOrg?: boolean;
   public isExpired?: boolean | 'unknown';
+  public password?: string;
```

Comment:
is password really optional? Seems to be set in the constructor right?

Reply:
Should be. I will remove init from ctor